### ISSUE TITLE
Setting to suppress missing documentation messages for Access <> Public objects

### ISSUE THREAD

**Issue by rvanbekkum** [365businessdev/vscode-alxmldocumentation#40]
Hi @365businessdev,
Could you add a setting that let's you specify that you would only like to see "missing documentation"-messages from the code-analyzer if the object is part of the public API? (So, no message if `Access = Internal`, for example)
If this is already possible, then could you please let me know how? 😃

**Comment by sirhc101** [comment]
Related to reopened issue #28

**Comment by sirhc101** [comment]
Hi @rvanbekkum,

Thank you very much for reporting. This is a bug in our RegEx to detect object access level!
I fixed this in version 1.0.3.

Basically the documentation was wrong. The configuration parameter `CheckProcedureDocumentationForAccessLevel` is exactly handling this. So with the new version procedures will not be checked in objects with "Access" other then "Public".  
At least in default configuration:
```json
    "al-xml-doc.CheckProcedureDocumentationForType": [
        "Global Procedures",
        "Local Procedures",
        "Internal Procedures",
        "Protected Procedures",
        "Event Publisher"
    ],
    "al-xml-doc.CheckProcedureDocumentationForAccessLevel": [
        "Public"
    ]
```

By the way: If you add `Internal` to `CheckProcedureDocumentationForAccessLevel` but remove `Internal Procedures` from `CheckProcedureDocumentationForType` you will also check `Internal` objects but not `internal procedures`.

I know it's not perfect from handling perspective right now. We're thinking for a better way to manage this, but for now there is no better idea...
